routes/swagger.yaml

```diff
@@ -6,10 +6,9 @@ info:
   description: The REST API for clearlydefined.io
 
 servers:
-  - url: https://api.clearlydefined.io
```

Comment:
Not sure why you have removed the endpoints?
P.S. we should fix the dev.clearlydefined.io one and change it to dev-api.clearlydefined.io